confluent_kafka/src/Consumer.c

```diff
@@ -988,7 +1037,7 @@ static PyMethodDef Consumer_methods[] = {
 	  "  .. note: Callbacks may be called from this method, "
 	  "such as ``on_assign``, ``on_revoke``, et.al.\n"
 	  "\n"
-	  "  :param float timeout: Maximum time to block waiting for message, event or callback.\n"
+	  "  :param float timeout: Maximum time to block waiting for message, event or callback. (Seconds)\n"
```

Comment:
I like the idea of including units in the parameter name (even if the convention for timeouts in the language is seconds). `secondsTimeout` is what I'd probably go with, though it may be a bit unusual. Not expecting you to change these, just saying. Much more effective than docs.

Reply:
I believe that is atypical for Python, and we can't change a named parameter anyway, so no luck, Matt :)